app/lib/theme/widgets/DueDateWidget.tsx

```diff
@@ -0,0 +1,92 @@
+import { WidgetProps } from "@rjsf/core";
+import React, { forwardRef, useRef } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday";
+import getTimestamptzFromDate from "lib/helpers/getTimestamptzFromDate";
+import { DateTime, Interval } from "luxon";
+
+const DueDateInput = forwardRef<HTMLDivElement, WidgetProps>(
```

Comment:
Any reason we use a `forwardRef` here, and not a regular `React.FC<WidgetProps>` or `React.FC<Partial<WidgetProps>>` ? is it required by the react-datepicker library?

Reply:
It is required by `react-datepicker` unfortunately. The `onClick` needs to get forwarded from the datepicker to the input.

Reply:
Ah, makes sense. 👍